IntellitectTerminal.Web/src/views/Home.vue

```diff
@@ -2,466 +2,63 @@
-  doc: HTMLInputElement | null = null;
-
-  // The stored string the user is typing
-  userInput: string = "";
-
-  // Entire file system
-  fileSystemTree: TreeNode = new TreeNode("/", false, null, []);
-
-  // Path we are on
-  pathTree: TreeNode = new TreeNode("/", false, null, []);
-
-  // Hostname relative to path we are on
-  hostname = `[\x1b[34mintellitect\x1B[0m@localuser \x1b[34m${this.pathTree.Name}\x1b[0m]$ `;
-
-  // Present working directory
-  pwd: string = "/";
-
-  // Update the path we are on and display hostname correctly
-  updatePath(location: TreeNode) {
-
-    // Traverse up the tree until root to get the pwd.
-    this.pwd = "";
-    let traverse: TreeNode = location;
-    while (traverse.Parent != null) {
-      this.pwd = ("/" + traverse.Name) + this.pwd;
-      traverse = traverse.Parent;
-    }
-
-    this.pathTree = location;
-    this.hostname = `[\x1b[34mintellitect\x1B[0m@localuser \x1b[34m${location.Name}\x1b[0m]$ `;
-  }
-
-  // Position the cursor is currently at. This is needed for back spaces.
-  cursorPosition = this.hostname.length;
-  term = new Terminal({
-    cursorBlink: true,
-    fontSize: 30,
-    cols: 200,
-    fontFamily: "monospace",
-  });
-
-  history = [];
-  welcomeMessage = `\n\nWelcome to the \x1b[34mIntelliTect Terminal\x1b[0m! View commands by typing help.\n\n`;
-
+  
   async created() {
 
     // XTerms input
-    const input = document.getElementById('terminal');
+    const xtermElement = document.getElementById('terminal');
 
     // Get the file input element that is hidden in index.html
-    this.doc = document.getElementById("file") as HTMLInputElement;
+    const fileUploader = document.getElementById("file") as HTMLInputElement;
 
-    // On doc input
-    this.doc?.addEventListener("change", (files) => {
+    if (xtermElement != null) {
 
-      // Create the http request
-      // TODO: Bug with submitting the same file after request
-      let formData = new FormData();
-      formData.append("file", this.doc!.files![0]);
-      formData.append("userId", this.user!.userId!);
-      fetch("/api/CommandService/SubmitFile", {
-        method: "POST",
-        body: formData
-      }).then((response) => response.json()).then((result) => {
-        console.log('Success:', result);
-        output("submit", this.term, "File submitted. Use \x1b[31mverify\x1b[0m to confirm the submission.")
-        this.term.write(this.hostname);
-      }).catch((error) => {
-        console.error('Error:', error);
-        output("submit", this.term, "An error occured uploading the file.");
-        this.term.write(this.hostname);
-      });
-    })
-    if (input != null) {
-       // if user is in cookies, get user id from cookies
-      let userId = getCookie('userId');
-      let user;
-      if (userId == null) {
+      const api: API = {
+        command: new CommandServiceViewModel(),
+        user: new UserServiceViewModel()
+      }
+
+       let userId = getCookie('userId');
```

Comment:
```suggestion
       let userId = Cookies.getCookie('userId');
```
I think this is needed to fix the cookie issue